### ISSUE TITLE
Move the portal from demo to operational site

### ISSUE THREAD

**Issue by janeuhd** [COATnor/COAT_portal_issues#75]
Deadline for moving the portal to its final address is Jan. 31st 2020.

**Comment by frafra** [comment]
The new version has been finished on Jan 30th 2020, but testing of #74 from both sides would be preferred.

**Comment by frafra** [comment]
There is a delay related to some infrastructure problems that should be fixed soon hopefully (~1 week).

**Comment by frafra** [comment]
coat.nina.no (operational site) and coatdemo.nina.no (test site) are up and running.

**Comment by janeuhd** [comment]
@frafra Could you explain again how I use the demo vs the operational? 
Is it so that the demo can be used freely as a sandbox? 
Will any changes we wish to make be made to both versions, or will the new demo be left as it is now, while we change the operational towards the finished product?
Will all the users be migrated to the new demo and the operational from the 'frafrademo', or will we need to establish them again?

I have a few layout/structural changes to suggest which I will put in a separate issue.

**Comment by frafra** [comment]
@janeuhd sure!

What follows is a list of suggestions on how I think the systems should be used.

- https://coatdemo.nina.no can be used as a sandbox. You should consider that a volatile storage system, like the Kladd folder for NINA users. 
- https://coat.nina.no is the official repository. My suggestion is to run automated procedures (like upload/changes) on that only after they have been tested on coatdemo.

The two system are entirely independent in terms of storage, data, users and settings. The software, instead, will be kept the same on both on those instances, but if we have new major features, we will deploy on coatdemo first, ask for your approval and then roll out on the official coat repository.

I would keep this separation between the two system for the entire life of the project.

Given that coatdemo should be considered as a volatile repository and the various changes that have been made, I would suggest against moving data from the old coatdemo, but it could be done if needed.

It would be easier for the moment to recreate the users on the new coatdemo, but we can consider moving them if required.

**Comment by janeuhd** [comment]
thanks for clarifying!

**Comment by frafra** [comment]
I think this issue can be closed, but we can add the difference between testing and production to the documentation.
